### PR TITLE
[Accessibility][Reflow] Error analysis tree map & Data explorer & Feature importance Reflow

### PR DESCRIPTION
**Pull request by RubyZ10**
## Description

**Note: 2D information is not required for reflow, such as error analysis tree**
Error analysis tree map
image
image
image

Data explorer
image
image
image
image

Feature Importance
image
image
image
image
image
image
image
image
image

## Checklist

- [x] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/WhatIfTab.styles.ts [hunk 1]
```diff
@@ -128,14 +132,20 @@ export const whatIfTabStyles: () => IProcessedStyleSet<IWhatIfTabStyles> =
       choiceBoxArea: {
         alignItems: "baseline",
         display: "flex",
-        flexDirection: "row"
+        flexDirection: "row",
+        flexFlow: "wrap"
       },
       choiceGroup: {
-        paddingLeft: "30px"
+        // paddingLeft: "30px"
```
Comment (tongyu-microsoft) [inline libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/WhatIfTab.styles.ts:139]: Is this comment needed? Same as comment below

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/utils.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { FeatureImportancesTabOptions } from "../FeatureImportances";
+
+import { ITabsViewProps } from "./TabsViewProps";
+
+export function getInfo(
+  tabKey: GlobalTabKeys,
+  props?: ITabsViewProps,
+  errorAnalysisOption?: ErrorAnalysisOptions,
+  featureImportanceOption?: FeatureImportancesTabOptions
+): string[] {
```
Comment (tongyu-microsoft) [inline libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/utils.ts:17]: what about create an interface with `{body: string; title: string}`? So we can replace `getInfo(...)[0]` with `getInfo(...).body`, which is easier to read?

**Review by tongyu-microsoft — APPROVED** [review]

File: libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/utils.ts (hunk 2 above)
Comment (RubyZ10) [reply libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/utils.ts]: updated, thanks!

File: libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/WhatIfTab.styles.ts (hunk 1 above)
Comment (RubyZ10) [reply libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/WhatIfTab.styles.ts]: removed both, thanks!

**Pull request merged**